### PR TITLE
android/*: add pt_PT translations

### PR DESCRIPTION
**Pull request by joao-conde**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.pt_PT/android/am.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+- Iniciar uma atividade e passar-lhe dados:
+
+`am start -a {{android.intent.action.VIEW}} -d {{tel:123}}`
+
+- Iniciar uma atividade com uma ação e categoria específicas:
+
+`am start -a {{android.intent.action.MAIN}} -c {{android.intent.category.HOME}}`
+
+- Converter um intent num URI:
```
Comment (waldyrious) [inline pages.pt_PT/android/am.md:18]: Sendo "intent" uma keyword específica neste context, eu sugeriria formatar de acordo:

```suggestion
- Converter um `intent` num URI:
```

File: pages.pt_PT/android/bugreport.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# bugreport
+
+> Mostra um relatório de bugs do Android.
+> Este comando só pode ser utilizado com a `adb shell`.
+> Mais informações: <https://cs.android.com/android/platform/superproject/+/master:frameworks/native/cmds/bugreport>.
+
+- Mostra um relatório completo de bugs de um dispositivo Android:
```
Comment (waldyrious) [inline pages.pt_PT/android/bugreport.md:7]:
```suggestion
- Mostrar um relatório completo de bugs de um dispositivo Android:
```

File: pages.pt_PT/android/bugreportz.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# bugreportz
+
+> Gera um relatório de bugs do Android em formato .zip.
+> Esse comando só pode ser utilizado com a `adb shell`.
+> Mais informações: <https://cs.android.com/android/platform/superproject/+/master:frameworks/native/cmds/bugreportz>.
+
+- Mostra um relatório completo de bugs de um dispositivo Android em formato .zip:
```
Comment (waldyrious) [inline pages.pt_PT/android/bugreportz.md:7]:
```suggestion
- Mostrar um relatório completo de bugs de um dispositivo Android em formato .zip:
```

(e o mesmo nos exemplos abaixo)

File: pages.pt_PT/android/cmd.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# cmd
+
+> Gestor de serviços do Android (service manager).
```
Comment (waldyrious) [inline pages.pt_PT/android/cmd.md:3]: Que tal assim?

```suggestion
> Gestor de serviços (service manager) do Android.
```

File: pages.pt_PT/android/cmd.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+> Mais informações: <https://cs.android.com/android/platform/superproject/+/master:frameworks/native/cmds/cmd/>.
+
+- Lista todos os serviços em execução:
```
Comment (waldyrious) [inline pages.pt_PT/android/cmd.md:6]:
```suggestion
- Listar todos os serviços em execução:
```

(e o mesmo nos exemplos abaixo)

File: pages.pt_PT/android/dalvikvm.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# dalvikvm
+
+> A máquina virtual Java do Android.
+> Mais informações: <https://source.android.com/devices/tech/dalvik>.
+
+- Inicia um programa Java:
```
Comment (waldyrious) [inline pages.pt_PT/android/dalvikvm.md:6]:
```suggestion
- Iniciar um programa Java:
```

File: pages.pt_PT/android/am.md (hunk 1 above)
Comment (joao-conde) [reply pages.pt_PT/android/am.md:18]: 👍

File: pages.pt_PT/android/getprop.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# getprop
+
+> Obtém informações sobre propriedades do sistema Android (system props).
```
Comment (waldyrious) [inline pages.pt_PT/android/getprop.md:3]: Mesma sugestão que fiz para o `cmd`:

```suggestion
> Obtém informações sobre propriedades do sistema (system props) Android.
```

File: pages.pt_PT/android/input.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+- Envia um único toque para um dispositivo Android:
+
+`input tap {{x_pos}} {{y_pos}}`
+
+- Envia um gesto de deslizar para um dispositivo Android:
+
+`input swipe {{x_inicio}} {{y_inicio}} {{x_fim}} {{y_fim}} {{duração_em_ms}}`
+
+- Envia um toque duradouro usando um gesto de deslize para um dispositivo Android:
```
Comment (waldyrious) [inline pages.pt_PT/android/input.md:23]: "Duradouro" soa-me ligeiramente anacrónico, e um pouco ambíguo também. O que achas de "prolongado"?

File: pages.pt_PT/android/logcat.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
+# logcat
+
+> Exibe um conjunto de mensagens de sistema, incluindo a stack de execução do programa quando um erro ocorreu e mensagens de informação criadas por aplicações.
```
Comment (waldyrious) [inline pages.pt_PT/android/logcat.md:3]: Uma pequena sugestão de refraseamento:

```suggestion
> Exibe um conjunto de mensagens de sistema, incluindo a stack de execução do programa  em caso de erro, e mensagens de informação criadas por aplicações.
```

File: pages.pt_PT/android/input.md (hunk 8 above)
Comment (joao-conde) [reply pages.pt_PT/android/input.md:23]: Soa melhor 👍

**Comment by waldyrious** [comment]
Muito obrigado pela contribuição, @joao-conde! E bem-vindo ao tldr-pages, espero que continues por aqui! :smiley:

Deixei algumas sugestões inline, mas não repliquei o comentário de usar o infinitivo em todas as páginas, para não ser repetitivo. Podes por favor fazer essa alteração em todas as páginas do PR? Thanks!

**Comment by joao-conde** [comment]
[waldyrious](https://github.com/waldyrious) good review suggestions, applied them and made use of same temporal verb form everywhere in the folder

File: pages.pt_PT/android/wm.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+
+> Exibe informações da tela de um dispositivo Android
+> Esse comando só pode ser usado através da `adb shell`.
+> Mais informações: <https://adbinstaller.com/commands/adb-shell-wm-5b672b17e7958178a2955538>.
+
+- Mostrar o tamanho da tela de um dispositivo Android:
+
+`wm {{size}}`
+
+- Mostrar a densidade de pixeis da tela de um dispositivo Android:
```
Comment (waldyrious) [inline pages.pt_PT/android/wm.md:11]: Em português, "pixeis" (com i) tem acento (ver [aqui](https://translatewiki.net/wiki/Portal:Pt/Cheatsheet/pt#pixels) e [aqui](https://pt.wiktionary.org/wiki/p%C3%ADxel)):

```suggestion
- Mostrar a densidade de píxeis da tela de um dispositivo Android:
```

Comment (joao-conde) [reply pages.pt_PT/android/wm.md:11]: My bad!

File: pages.pt_PT/android/wm.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
+# wm
+
+> Exibe informações da tela de um dispositivo Android
+> Esse comando só pode ser usado através da `adb shell`.
```
Comment (waldyrious) [inline pages.pt_PT/android/wm.md:4]: ...e aqui também:

```suggestion
> Este comando só pode ser usado através da `adb shell`.
```

File: pages.pt_PT/android/wm.md (hunk 10 above)
Comment (waldyrious) [reply pages.pt_PT/android/wm.md:11]: Not at all, that's a common confusion since it's a loanword :)

**Comment by waldyrious** [comment]
@navarroaxel unrelated to this PR, but we need to fix the [GH actions job](https://github.com/tldr-pages/tldr/actions/runs/3226371313/jobs/5279810245) to not spell-check content outside the (English) `pages/` directory. Has that been identified?

**Review by waldyrious — APPROVED** [review]
LGTM, very nice work @joao-conde!

**Review by navarroaxel — APPROVED** [review]

**Comment by waldyrious** [comment]
@navarroaxel / @kbdharun, can you say something about my comment above? I'm glad you agree, but what (if any) should be the next steps?

**Comment by navarroaxel** [comment]
I can check it later, otherwise @CleanMachine1 do you know how to exclude the root directory?

**Comment by kbdharun** [comment]
> we need to fix the [GH actions job](https://github.com/tldr-pages/tldr/actions/runs/3226371313/jobs/5279810245) to not spell-check content outside the (English) `pages/` directory. Has that been identified?

I think it already exists in the workflow not sure why it doesn't work. 

Reference: [Line 16 and 17](https://github.com/tldr-pages/tldr/blob/main/.github/workflows/codespell.yml)
```yml
 # Ignore all other languages except English
          files_ignore: pages.*/*/* 
```

> can you say something about my comment [above](https://github.com/tldr-pages/tldr/pull/8935#issuecomment-1274524090)? I'm glad you agree, but what (if any) should be the next steps?

I agree with you the workflow mustn't run on PRs on other pages than the main. I am still learning workflows with GitHub actions, so at the moment I can't provide a solution unfortunately. I will cc @CleanMachine1 let's see what he thinks.

**Comment by kbdharun** [comment]
>
> I don't really understand this syntax. thinking Is it documented somewhere?

I too can't find documentation on this, it was added in PR #8359.

**Comment by waldyrious** [comment]
@CleanMachine1 any insight on the above?

**Pull request merged**